build.gradle.kts

```diff
@@ -106,7 +106,7 @@ kotlin {
                 api(ktor("network-tls"))
                 implementation("fr.acinq.secp256k1:secp256k1-kmp-jni-jvm:$secpJniJvmVersion")
                 implementation("org.slf4j:slf4j-api:1.7.36")
-                api("org.xerial:sqlite-jdbc:3.32.3.2")
+                api("org.xerial:sqlite-jdbc:3.32.3.3")
```

Comment:
Let's take this opportunity to move the declaration to `jvmTest.dependencies`, where it belongs, and use `implementation()` instead of `api()`.

This is a vestige of the past when we had a default database implementation in `commonMain`.